bodhi/server/scripts/check_policies.py

```diff
@@ -35,9 +35,18 @@ def check():
     initialize_db(config.config)
     session = Session()
 
-    updates = models.Update.query.filter(models.Update.status.in_(
-        [models.UpdateStatus.pending, models.UpdateStatus.testing])).order_by(
-        models.Update.id.desc())
+    updates = models.Update.query.filter(
+        models.Update.status.in_(
+            [models.UpdateStatus.pending, models.UpdateStatus.testing])
+    ).filter(
+        models.Update.release_id == models.Release.id
+    ).filter(
+        models.Release.state.in_(
+            [models.ReleaseState.current, models.ReleaseState.pending])
+    ).order_by(
+        models.Update.id.desc()
```

Comment:
The most recent updates are the most likely to be untested. By checking those first, you increase the chances of getting the missing tests error from Greenwave - if we instead sort ascending, we move them to the end of the list we check and increase the chances of finding their tests.

Reply:
I didn't add this piece, but I'm ok with changing this.